TicTacToeApp/ViewController.swift

```diff
@@ -1,152 +1,99 @@
 
 import UIKit
 
-public class BoardButton: UIButton {
-}
-
-public class BoardView: UIView {
-
-  // hypothetical interface
-  public func enableInput() {}
-  public func disableInput() {}
-  public func clear() {}
-
-}
-
-// Refactoring alert dialog
-// Might be overkill. Let's revisit after board button refactorings
-//public class ConfirmResetGameAlert: UIAlertController {}
-// Alternatively, consider builder patter, or just wrapping the alert dialog
-//     i.e. composition over inheritance
-//public class ConfirmResetGameAlert {
-//  var alertController: UIAlertController?
-//}
-
 public class ViewController: UIViewController {
```

Comment:
Even though `ViewController` is the generated name, it is conventional to name the class something more specific. For instance, what would happen if you needed to have a second controller?